packages/scaffold/src/modal/w3m-modal/index.ts

```diff
@@ -189,16 +187,6 @@ export class W3mModal extends LitElement {
         return
       }
 
-      // If the network has changed and signOnNetworkChange is enabled, sign out
```

Comment:
What if the user change the network from the wallet when the w3m is closed state? I guess this is the only place that will trigger SIWE 🤔